backends/p4tools/CONTRIBUTORS.md

```diff
@@ -0,0 +1,18 @@
+# Overview 
+
+P4Testgen is a test oracle for the P4 language. Given a P4_16 program and a specification of the underlying architecture, it automatically generates a comprehensive set of input/output tests that can be executed to validate a target device.
```

Comment:
This is specific to P4Testgen, not P4Tools.

Reply:
I think that's fine for now...